recipes/r-secutrialr/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = "1.0.0" %}
+{% set github = "https://github.com/SwissClinicalTrialOrganisation/secuTrialR" %}
+
+package:
+  name: r-secutrialr
+  version: "{{ version }}"
+
+source:
+  url: "{{ github }}/archive/{{ version }}.tar.gz"
+  sha256: 9dae54b415cf94ed1d3a386ea60415dacaaa7737a17e37e9aa424e668ddaab46
+
+build:
+  noarch: generic
+  number: 0
+  script: 
+    - $R CMD INSTALL --build .            # [not win]
+    - "\"%R%\" CMD INSTALL --build ."  # [win]
+
+requirements:
+  host:
+    - r-base
+    - r-haven >=2.2.0
+    - r-readr
+    - r-readxl
+    - r-tcltk2
+    - r-igraph
+    - r-stringr
+    - r-tibble
+    - r-magrittr
+    - r-purrr
+    - r-tidyr
+    - r-dplyr
+    - r-rlang
+    - r-lubridate
+    - r-knitr
+    - r-rmarkdown
+    - r-testthat
+    - r-tufte
+    - r-lintr
+  run:
+    - r-base
+    - r-haven >=2.2.0
+    - r-readr
+    - r-readxl
+    - r-tcltk2
+    - r-igraph
+    - r-stringr
+    - r-tibble
+    - r-magrittr
+    - r-purrr
+    - r-tidyr
+    - r-dplyr
+    - r-rlang
+    - r-lubridate
+    - r-knitr
+    - r-rmarkdown
+    - r-testthat
+    - r-tufte
+    - r-lintr
+
+test:
+  commands:
+    - $R -e "library('secuTrialR')"       # [not win]
+    - "\"%R%\" -e \"library('secuTrialR')\""  # [win]
+
+about:
+  home: https://swissclinicaltrialorganisation.github.io/secuTrialR/
+  dev_url: "{{ github }}"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
+  summary: Handling of Data from the Clinical Data Management System secuTrial 
+
+extra:
+  recipe-maintainers:
```

Comment:
Would be good to have `conda-forge/r` as a maintainer, too. I'll add them once the feedstock is generated.

Reply:
Thank you, its appreciated.